app/views/devise/invitations/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="grid-row">
+  <div class="column-full">
+    <%= form_for resource,
+                 as: resource_name,
+                 url: invitation_path(resource_name),
+                 html: { method: :put } do |f| %>
+
+       <h1 class="heading-large">
+         <%= t(".heading") %>
+       </h1>
+
+       <p><%= t(".paragraph_1") %></p>
+
+      <%= render("waste_carriers_engine/shared/errors", object: resource) %>
+
+      <%= f.hidden_field :invitation_token %>
+
+      <% if f.object.class.require_password_on_accepting %>
```

Comment:
Do we need to check this in our system? I don't think we will have any instance where this is going to be false?

Reply:
Good point, will remove that.

Reply:
Fixed! https://github.com/DEFRA/waste-carriers-back-office/pull/574/commits/8eb3374a1b2282b071ace36afde84a5c0d331aae